packages/documentation/src/content/docs/overview/concepts/accounting.mdx

```diff
@@ -1,139 +1,261 @@
+<Steps>
+1. Rafiki receives packets worth 10 EUR. Your EUR liquidity increases to 20 (10 + 10).
+2. The current EUR-to-USD exchange rate is applied, with 10 EUR equating to 12 USD.
+3. Rafiki sends packets worth 12 USD. Your USD liquidity decreases to 38 (50 - 12).
+</Steps>
 
-### Assets
+**Cross-currency transaction #2:**
 
-An asset represents an item of value that can be transferred via the Interledger Protocol. Since the Interledger Protocol aims to create an internet of value, it allows for the transfer of any asset, not just currency. In practice, however, assets are usually denominated in a currency (fiat or branded currencies).
+<Steps>
+1. Rafiki receives packets worth 50 EUR. Your EUR liquidity increases to 70 (20 + 50).
+2. The current EUR-to-USD exchange rate is applied, with 50 EUR equating to 55 USD.
+3. The transaction fails. Since your USD liquidity is 38, you do not have enough liquidity to cover the transaction.
+4. Your EUR liquidity reduces back to 20.
+</Steps>
+</details>
 
-In Rafiki, the `asset` type is comprised of the following properties.
+#### Peer liquidity accounts
 
-<div class="overflow-table">
+Peer liquidity is the credit line you've extended to a peer. A peer liquidity account represents the amount of the line of credit that the peer still has available to them. You have one liquidity account for each peer and the account is denominated in the asset you both agreed to transact in.
 
-| Property     | Type    | Description                                                                                                            | Example |
-| ------------ | ------- | ---------------------------------------------------------------------------------------------------------------------- | ------- |
-| `value `     | BigInt  | A numerical amount                                                                                                     | `10000` |
-| `assetCode`  | String  | Should be an <LinkOut href="https://en.wikipedia.org/wiki/ISO_4217">ISO 4217 currency code</LinkOut> whenever possible | `"USD"` |
-| `assetScale` | Integer | Difference in order of magnitude between the standard unit and a fractional unit                                       | `2`     |
+The amount of credit that you extend to a peer, the asset that you transact in, and the mechanism you use to settle are just a few items that should be defined in your respective peering agreements.
+
+:::note
+A peering agreement is a legal contract between the parties involved in a peering relationship. It defines terms such as the assets involved, auth tokens, connection endpoints, and other operational details. It is not configured or managed within Rafiki but is necessary for establishing the terms under which assets are exchanged.
+:::
+
+If a peer’s liquidity is insufficient (e.g., they’ve used up their allotted credit line), payments will not be processed. Your peer should settle with you so that you can reset their liquidity.
```

Comment:
Maybe we can mention you can set up thresholds for peer (and asset accounts) to get alerted when liquidity drops below a certain amount